packages/twenty-front/src/hooks/useNavigateApp.ts

```diff
@@ -0,0 +1,20 @@
+import { AppPath } from '@/types/AppPath';
```

Comment:
we avoid putting things into src/hooks overall,
could we move this to modules/navigation?